TedTalks/TedTalks/Controllers/TedTalkListViewController.swift

```diff
@@ -8,21 +8,32 @@
 import UIKit
 
 class TedTalkListViewController: UIViewController {
+    
 
     @IBOutlet weak var tableView: UITableView!
     
+    @IBOutlet weak var tedTalksSearchBar: UISearchBar!
+    
+    @IBOutlet weak var tedTalksFilterPickerView: UIPickerView!
+    
+    private var pickerData: [String] = ["All filters", "Event", "Main speaker", "Title", "Name", "Description"]
```

Comment:
This is a suggested improvement, we could define this filters as an `Enum` that way we avoid the risk of using string literals in several places of the App, in this case you use this same filters in `TedTalkManager` so any change on this will not only mean you'll need to update in several places with the risk of typos but also that you can forget to handle a filter that you should since there's no type enforcement to handle all cases.

Reply:
Thank you! Fixed.